README.md

```diff
@@ -1,12 +1,75 @@
-# Talk Timer
+<p align="center">
+  <h1 align="center">TALK TIMER</h1>
+</p>
 
-A simple timer for talks, with configurable time for warning and end of talk.
+<h3 align="center">A simple timer for talks, with configurable time for warning and end of talk.</h1>
 
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app). It uses [shadcn/ui](https://ui.shadcn.com/) for the UI components.
+<div>
+  <img src="./docs/Images/HomePage.png" alt="talktimer-home" />
+</div>
 
-## Getting Started
+  <h2 id="About-project" align="center">⚡️ About the Project</h2>
 
-First, run the development server:
+<div>
+  <p>
+    This is a simple timer that can be used during speech or competition to mark timings. The default configuration is:
+    <ul>
+      <li>Title: My Lightning Talk ⚡</li>
+      <li>Yellow Threshold: 90</li>
+      <li>Red Threshold: 120</li>
+    </ul>
+  </p>
+  <img src="./docs/Images/Selection.png">
+</div>
+<div>
+  <p>When the timer reaches the Yellow threshold, the color is immediately changed to Yellow</p>
+  <img src="./docs/Images/Yellow.png">
+</div>
+<div>
+  <p>When the timer reaches the Red threshold, it means the time is almost up. Manually uou would need to pause it and refresh it</p>
+  <img src="./docs/Images/Red.png">
+</div>
+<br />
+
+  <h2 id="tech-stack" align="center">⚡️ Tech Stack</h2>
+
+### Frontend Technology
+NextJS + shadcn/ui
+
+### Deployment Platform
+Vercel
```

Comment:
Deployment can be done on any platform, its not just limited to Vercel. It has been deployed to vercel currently.